client/sttp-client/src/main/scalajvm/sttp/tapir/client/sttp/SttpClientInterpreterPlatformSpecific.scala

```diff
@@ -0,0 +1,34 @@
+package sttp.tapir.client.sttp
+
+import sttp.client3.HttpURLConnectionBackend
+import sttp.model.Uri
+import sttp.tapir.Endpoint
+
+trait SttpClientInterpreterPlatformSpecific {
+
+  def toBlockingClientUnsafe[I, E, O](e: Endpoint[I, E, O, Any], baseUri: Uri)(implicit
```

Comment:
Again naming :) In sttp-client, there's the concept of the `sttp.client.quick` package, which contains the backend instance. So maybe here we could follow suit and have `toClientQuick` etc?